### DISCUSSION TITLE
Terminal on launch

### DISCUSSION THREAD

**Discussion by sharypovandrey** [coder/code-server#2273] [General]
Hello guys,

1) Is there a way to open a terminal on the code-server launch?
2) can I specify a command on the terminal launch?

**Comment by code-asher** [comment]
I don't think VS Code supports this, at least as far as I'm aware. You
could write a plugin to do this, though. I think that would probably be
the best way to implement it.

**Comment by bpmct** [comment]
@sharypovandrey I haven't tried this, but it looks like you can try it with "tasks" in VS Code and just launch the task to open the terminal on `folderOpen`: https://code.visualstudio.com/docs/editor/tasks#_run-behavior
